CONTRIBUTING.rst

```diff
@@ -0,0 +1,62 @@
+Contributing
+=========================
+
+As an open source project, Mesa welcomes contributions of many forms.
+
+In no particular order, examples include:
+
+- Code patches
+- Bug reports and patch reviews
+- New features
+- Documentation improvements
+- Tutorials
+
+No contribution is too small. Although, contributions can be to big, so let's discuss via the `email list`_ OR `an issue`_.
+
+**To submit a contribution**
+
+- Create a ticket for the item that you are working on.
+- Fork the Mesa repository
+- Create a new branch if you aren't contributing to an existing branch.
+- Edit the code.
+- Use `PEP8`_ and the `Google Style Guide`_ as the coding standards for Python.
+- If implementing a new feature, include some documentation and ideally a working example or unit test as well.
+- Submit as pull requests
+- Describe the change w/ ticket number(s) that the code fixes
+
+.. _`email list` : https://groups.google.com/forum/#!forum/projectmesa
+.. _`an issue` : https://github.com/projectmesa/mesa/issues
+.. _`PEP8` : https://www.python.org/dev/peps/pep-0008
+.. _`Google Style Guide` : https://google-styleguide.googlecode.com/svn/trunk/pyguide.html
+
+
+Testing
+--------
+
+.. image:: https://coveralls.io/repos/projectmesa/mesa/badge.svg
+    :target: https://coveralls.io/r/projectmesa/mesa
+
```

Comment:
I thought that was already captured in the text. I don't want to oversell folks on our tests, because we are still lacking quite a bit. 

Can you suggest a specific edit to the following?

```
Testing
--------

.. image:: https://coveralls.io/repos/projectmesa/mesa/badge.svg
    :target: https://coveralls.io/r/projectmesa/mesa

We are continually working to improve our testing. At the moment, we've been testing features by implementing them in simple models. This is useful since it also expands the library of sample models.
```